lms/djangoapps/courseware/models.py

```diff
@@ -400,3 +400,34 @@ class Meta(object):
         default=False,
         help_text=_('Disable the dynamic upgrade deadline for this course run.')
     )
+
+
+class CourseScheduleConfiguration(ConfigurationModel):
```

Comment:
I'm not sure? I don't love this config model... I'm thinking we should back out all of the changes associated with it and re-evaluate the policy logic.